### ISSUE TITLE
sails.config.globals missing 'adapters'

### ISSUE THREAD

**Issue by tarlepp** [balderdashy/sails-docs#451]
I just noticed that globals configuration is missing adapters. I have disabled all but ```sails``` in this file and didn't see any reference to this ```adapters```.

```
sails> console.log(sails.config.globals);
{ _: false,
  async: false,
  sails: true,
  services: false,
  models: false,
  adapters: true }
```

**Comment by sailsbot** [comment]
Thanks for posting, @tarlepp.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by tarlepp** [comment]
Hmm, cannot re-open this, the issue is still open.
